modules/rosa-sts-creating-roles.adoc

```diff
@@ -0,0 +1,624 @@
+                 "route53:ListHostedZonesByName",
+                 "route53:ListResourceRecordSets",
+                 "route53:ListTagsForResource",
+                 "route53:UpdateHostedZoneComment",
+                 "s3:CreateBucket",
+                 "s3:DeleteBucket",
+                 "s3:DeleteObject",
+                 "s3:GetAccelerateConfiguration",
+                 "s3:GetBucketAcl",
+                 "s3:GetBucketCORS",
+                 "s3:GetBucketLocation",
+                 "s3:GetBucketLogging",
+                 "s3:GetBucketObjectLockConfiguration",
+                 "s3:GetBucketRequestPayment",
+                 "s3:GetBucketTagging",
+                 "s3:GetBucketVersioning",
+                 "s3:GetBucketWebsite",
+                 "s3:GetEncryptionConfiguration",
+                 "s3:GetLifecycleConfiguration",
+                 "s3:GetObject",
+                 "s3:GetObjectAcl",
+                 "s3:GetObjectTagging",
+                 "s3:GetObjectVersion",
+                 "s3:GetReplicationConfiguration",
+                 "s3:ListBucket",
+                 "s3:ListBucketVersions",
+                 "s3:PutBucketAcl",
+                 "s3:PutBucketTagging",
+                 "s3:PutEncryptionConfiguration",
+                 "s3:PutObject",
+                 "s3:PutObjectAcl",
+                 "s3:PutObjectTagging",
+                 "sts:AssumeRole",
+                 "sts:AssumeRoleWithWebIdentity",
+                 "sts:GetCallerIdentity",
+                 "tag:GetResources",
+                 "tag:UntagResources"        
+            ],
+            "Resource": "*"
+        }
+    ]
+}
+EOM
+----
+====
+
+. Attach the inline policy in the [filename]`ManagedOpenShift_IAM_Role_Policy.json` file to the `ManagedOpenShift-IAM-Role` instance profile role.
++
+[source,terminal]
+----
+$ aws iam put-role-policy --role-name ManagedOpenShift-IAM-Role --policy-name ManagedOpenShift-IAM-Role-Policy --policy-document file://ManagedOpenShift_IAM_Role_Policy.json
+----
+
+== Control plane node instance profile role
+
+.Procedure
+. Write the trust policy for the `ManagedOpenShift-ControlPlane-Role` control plane node instance profile role to the  [filename]`ManagedOpenShift_ControlPlane_Role.json` file.
++
+.Command to write the trust policy
+[%collapsible]
+====
+[source,terminal]
+----
+$ cat << EOM > ManagedOpenShift_ControlPlane_Role.json
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Principal": {
+        "Service": "ec2.amazonaws.com"
+      },
+      "Action": "sts:AssumeRole"
+    }
+  ]
+}
+EOM
+----
+====
+. Create the `ManagedOpenShift-ControlPlane-Role` role with the trust policy using the AWS CLI.
++
+[source,terminal]
+----
+$ aws iam create-role --role-name ManagedOpenShift-ControlPlane-Role --assume-role-policy-document file://ManagedOpenShift_ControlPlane_Role_Policy.json [--permissions-boundary ${permissions_boundary_arn}]
```

Comment:
The file created in above command is ManagedOpenShift_ControlPlane_Role.json
But
the file used in this command is file://ManagedOpenShift_ControlPlane_Role_Policy.json
It is not runable